frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/validation/AuthAppCodeProcessor.java

```diff
@@ -75,7 +77,12 @@ public Optional<ErrorResponse> validateCode() {
             return Optional.of(ErrorResponse.ERROR_1042);
         }
 
-        incrementRetryCount(MFAMethodType.AUTH_APP);
+        if (!nonRegistrationJourneyTypes.contains(codeRequest.getJourneyType())
```

Comment:
just want to review this before it's merged

Reply:
the changes should only apply to the account creation email otp, not the 2nd factor